src/vivarium_census_prl_synth_pop/data/loader.py

```diff
@@ -343,7 +337,10 @@ def _read_and_format_raw_data(
     data_dir: Path, filenames: List[str], column_map: List[str], location: str
 ) -> pd.DataFrame:
     data = pd.concat(
-        [pd.read_csv(data_dir / file, usecols=column_map.keys()) for file in filenames]
+        [
+            pd.read_csv(data_dir / file, usecols=column_map.keys(), dtype={"SERIALNO": str})
+            for file in filenames
+        ]
     )
 
     data["SERIALNO"] = data["SERIALNO"].astype(str)
```

Comment:
Is this needed anymore given that you've set the dtype directly in the read_csv call?

Reply:
No; I missed this. I'll remove it.